SUMMARY.md

```diff
@@ -32,13 +32,13 @@
     * [Phenotype data](products/trait-data.md)
     * [Point Cloud Data](products/point-cloud-data.md)
   * [How to Access Data](user/how-to-access-data.md)
-    * [Using Clowder (Sensor and Genoomics data)](user/using-clowder.md)
-    * [Using Globus (Sensor and Genomics data)](user/using-globus.md)
-    * [Using BETYdb (trait data, experimental metadata)](user/using-betydb.md)
-       * [Accessing BETYdb via ArcMap and other GIS software](accessing-betydb-with-arcmap.md) 
-    * [Using CoGe (Genomics)](user/using-coge.md)
-    * [Using CyVerse (Genomics)](user/using-cyverse.md)
-    * [Using Analysis Workbench (all data)](user/using-analysis-workbench.md)
+    * [Using Clowder \(Sensor and Genoomics data\)](user/using-clowder.md)
```

Comment:
escaping the `()` appears to have no effect when compiled, but are the `\` necessary?

Reply:
Hmm, I didn't intentionally put those in. Not sure if they were already there or if GitBook put them in. Just noticed a typo there in "Genoomics"